start.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-docker run --rm -it -v ${PWD}:/usr/src/app/microsvc --name=pn-openapi-processor pagopa/pn-openapi-processor 
+docker run --rm -it -v ${PWD}:/usr/local/app/microsvc --name=pn-openapi-processor pagopa/pn-openapi-processor 
```

Comment:
Questo script, ora, è fuori sincrono rispetto al README.md

Suggerisco, nel README.md di mettere un link allo script in modo da avere una sola fonte di informazione.

Reply:
@marco-vit-pagopa aggiornato readme e start.sh